Debian/Dockerfile

```diff
@@ -166,6 +167,7 @@ RUN cpanm --installdeps .
 
 WORKDIR /srv
 RUN <<EOF
+# install Mail::JMAPTalk
```

Comment:
Wild, we build Mail::JMAPTalk in a way that ignores dependencies.

Reply:
Something to fix in the future, imho.